packages/web-components/src/global/_variables.scss

```diff
@@ -32,6 +30,7 @@ $color-gray-lightest: #f1f1f1;
 $color-gray-warm-dark: #494440;
 $color-gray-warm-light: #e4e2e0;
 $color-gray-cool-light: #dce4ef;
+$v3-color-error-dark: #b50909;
```

Comment:
Was it intentional not to generate this variables file so that it includes all of the other colors added to `packages/css-library/tokens/color.json`?

Reply:
Correct, you mentioned to only add the error variable to `web-components`

Reply:
Sorry I meant only adding the v3 colors we needed to the actual css-library (not just to this file that is generated by it). I don't think we should be manually updating this file at all.

I am open to adding all of those other colors to css-library if we feel like we are going to use them. I just wanted to avoid making the library larger than it needed to be pre-maturely.

Reply:
Ohhh, I see what you are saying. I can remove them all but that one and we can add as we need them then.